templates/frontend/operations_center/operations_center.html.twig

```diff
@@ -42,6 +43,7 @@
                 {% endif %}
                 {% include '@ForumifyPerscomPlugin/frontend/operations_center/tabs/courses.html.twig' %}
             {% endif %}
+            {% include '@ForumifyPerscomPlugin/frontend/operations_center/tabs/award-nominations.html.twig' %}
```

Comment:
The tab itself requires premium, but the tabpanel doesn't? Move tabpanel into the premium if 😅